### PR TITLE
chore: prepare 1.4.3 release and deprecate project

### LINKED ISSUE #674: Should deprecate intellij-tekton
We should deprecate this plugin because we're lacking the resources to tackle the current shortcomings. Without addressing those, there's little sense to keep supporting it:
* latest schema are not supported (`WhenExpression` replaced `Condition`, see [issue 668, comment-2014768254](https://github.com/redhat-developer/intellij-tekton/issues/668#issuecomment-2014768254)
* it cannot be run with the latest intellij-kubernetes, intellij-openshift-connector (to be able to do so we'd need to bump the kubernetes-client to the same version. We can't do this because there were changes in the schema, which needs effort to implements, see [issue 668, comment-2014768254](https://github.com/redhat-developer/intellij-tekton/issues/668#issuecomment-2014768254)

### PR DESCRIPTION
**Pull request by sbouchet**
fixes #674

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 [![License](https://img.shields.io/github/license/redhat-developer/intellij-tekton?style=for-the-badge)](https://github.com/redhat-developer/intellij-tekton/blob/main/LICENSE)
 
 # IntelliJ Tekton
+
+## NOTE: This project is no more maintained and marked as deprecated.
```
Comment (mohitsuman) [inline README.md:10]:
```suggestion
## NOTE: This plugin is no longer maintained and is officially deprecated.
```

File: src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <idea-plugin>
   <id>com.redhat.devtools.intellij.tekton</id>
-  <name>Tekton Pipelines by Red Hat</name>
+  <name>Tekton Pipelines by Red Hat (Deprecated)</name>
   <version>1.0</version>
   <vendor email="developers@redhat.com" url="https://www.redhat.com">Red-Hat</vendor>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:5]:
```suggestion
  <vendor email="devtools-team@redhat.com" url="https://www.redhat.com">Red-Hat</vendor>
```

Comment (mohitsuman) [reply src/main/resources/META-INF/plugin.xml:5]: I think having the devtools-team email is more useful than having developers@redhat.com (as this thing does not exist)

File: src/main/resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -12,6 +12,12 @@
     ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>1.4.3</b></p>
+  <ul>
+    <li>This extension is now deprecated.</li>
+    <li>Support IDEA IU-2024.1</li>
+    <li>Update Schemas tto Tekton Pipelines 0.56.0 and Triggers to 0.26.0</li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:19]:
```suggestion
    <li>Update Schemas to Tekton Pipelines 0.56.0 and Tekton Triggers to 0.26.0</li>
```

File: src/main/resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -12,6 +12,12 @@
     ]]></description>
 
   <change-notes><![CDATA[
+  <p><b>1.4.3</b></p>
+  <ul>
+    <li>This extension is now deprecated.</li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:17]: We do not need this line as we are mentioning at the top this plugin is deprecated.

**Review by mohitsuman — CHANGES_REQUESTED** [review]
wording updates

**Comment by sbouchet** [comment]
@mohitsuman 
fix reviews. please re-review.

**Review by adietish — APPROVED** [review]

**Review by mohitsuman — APPROVED** [review]
lgtm

**Pull request merged**